CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-cmake_minimum_required(VERSION 3.13...3.17)
+cmake_minimum_required(VERSION 3.13...3.30)
```

Comment:
Note that, this still needs to be changed to 3.27.7...3.30 to align with build.md, we can do it in another PR.